joplin/templates/wagtailadmin/pages/search_results.html

```diff
@@ -0,0 +1,57 @@
+{% load i18n wagtailadmin_tags %}
+<div>
+    {% if pages %}
+        <div class="coa-search-results-nav">
+
+          <!-- Saved for Joplin: if we want to show the COUNT of returned searches...
+          <!--
+          <h2>
+            {% blocktrans count counter=all_pages.count %}
+                There is one matching page
+            {% plural %}
+                There are {{ counter }} matching pages
+            {% endblocktrans %}
+          </h2>
+          -->
+
+          <!-- {% search_other %} -->
+
+          <nav class="listing-filter" aria-labelledby="page-types-title">
+              <h3 id="page-types-title" class="filter-title">{% trans "Page types" %}</h3>
+              <ul class="filter-options">
+
+                  {% if not selected_content_type %}
+                      <li style="background-color: #E6E6E6">All ({{ all_pages.count }})</li>
+                  {% else %}
+                      <li><a href="{% url 'wagtailadmin_pages:search' %}?q={{ query_string|urlencode }}">All ({{ all_pages.count }})</a></li>
+                  {% endif %}
+
+                  {% for content_type, count in content_types %}
+                      {% if content_type == selected_content_type %}
+                          <li style="background-color: #E6E6E6">{{ content_type.model_class.get_verbose_name }} ({{ count }})</li>
+                      {% else %}
+                          <li><a href="{% url 'wagtailadmin_pages:search' %}?q={{ query_string|urlencode }}&amp;content_type={{ content_type.app_label }}.{{ content_type.model|lower }}">{{ content_type.model_class.get_verbose_name }} ({{ count }})</a></li>
+                      {% endif %}
+                  {% endfor %}
+              </ul>
+          </nav>
+        </div>
+
+        <!--
+          This is where Joplin search results are modified for the UI for search results
+          on our search results page.
+        -->
+        {% include "wagtailadmin/pages/listing/_list_explore.html" with show_parent=0 sortable=1 sortable_by_type=0 full_width=1 ordering=request.GET.ordering show_ordering_column=1 orderable=1 %}
```

Comment:
Again, this file was brought up from the wagtail root lib so we could modify the headers in the search results UI. Here's where we can also add/remove column headers and behaviors